test/file-test.js

```diff
@@ -0,0 +1,26 @@
+var test = require('tape');
+var jsmidgen = require('../lib/jsmidgen.js');
+
+test('File -> setTicks should set the correct HDR_SPEED on valid input', function(t) {
+	var file  = new jsmidgen.File({
+		ticks: 1000
+	});
+	t.equal(file.ticks, 1000, 'ticks should be set to 1000');
```

Comment:
It would be nice to also add a test for a non-integer value of `ticks`, e.g. `133.7`.